docs/site/todo-list-tutorial-model.md

```diff
@@ -0,0 +1,115 @@
+---
+lang: en
+title: 'Add TodoList Model'
+keywords: LoopBack 4.0, LoopBack 4
+tags:
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/todo-list-tutorial-model.html
+summary: LoopBack 4 TodoList Application Tutorial - Add TodoList Model
+---
+
+### Models with relations
+
+A set of data can often be related to another; an entity may be able to provide
+access to another based on its relationship with the other entity. To take
+`TodoListApplication` one step further and establish relations with the existing
+`Todo` model as real-world applications often tend to do, we'll introduce the
+model `TodoList`.
+
+### Building a checklist for your Todo models
+
+A todo item is often grouped into a checklist along with other todo items so
+that it can be used to measure the progress of a bigger picture that the item is
+a part of.
+
+We'll create TodoList model to represent a checklist that contains multiple Todo
+items. Let's define TodoList model with the following properties:
+
+- a unique id
+- a title
+- a color to represent the TodoList with
+
+We can use the `lb4 model` command and answer the prompts to generate the model
+for us as follows:
+
+```sh
+$ lb4 model
+? Model class name: TodoList
+
+Let's add a property to TodoList
+Enter an empty property name when done
+
+? Enter the property name: id
+? Property type: number
+? Is ID field? Yes
+? Required?: No
+? Default value [leave blank for none]:
+
+Let's add another property to TodoList
+Enter an empty property name when done
+
+? Enter the property name: title
+? Property type: string
+? Required?: Yes
+? Default value [leave blank for none]:
+
+Let's add another property to TodoList
+Enter an empty property name when done
+
+? Enter the property name: color
+? Property type: string
+? Required?: No
+? Default value [leave blank for none]:
+
+Let's add another property to TodoList
+Enter an empty property name when done
+
+? Enter the property name:
+   create src/models/todo-list.model.ts
+   update src/models/index.ts
+
+Model TodoList was created in src/models/
+```
+
+Now that we have our new model, it's time to define its relation with the `Todo`
+model. To `TodoList` model, add in the following property:
+
+#### src/models/todo-list.model.ts
+
+```ts
+@model()
+export class TodoList extends Entity {
+  // ...properties defined by the CLI...
+
+  @hasMany(Todo) todos?: Todo[];
+
+  // ...constructor def...
+}
+```
+
+Don't forget to decorate your new relations property with `@hasMany`! As the
```

Comment:
Instead of saying don't forget ... I think it's better to say something like `Notice the `@hasMany()` decorator used to define this property, it lets LB4 know that a todo list can have many todo items`.

Reason for not saying don't forget is because this follows a code sample a user will likely copy and paste into their application.